lib/ansible/module_utils/cloud.py

```diff
@@ -67,34 +130,28 @@ def found(response_code):
         pass
 
     @classmethod
-    def backoff(cls, tries=10, delay=3, backoff=1.1):
-        """ Retry calling the Cloud decorated function using an exponential backoff.
+    def backoff(cls, backoff=default_backoff):
```

Comment:
I'd rather we not change the default behavior of `backoff` since there are custom modules (outside the Ansible tree) that use AWSRetry/CloudRetry and we consider module_utils to be public APIs.